glossary.yml

```
@@ -2735,13 +2791,11 @@
     def: >
       A test that checks whether the parts of a system work properly when put together.
 
-
-- slug: interpeter
-  en:
-    term: "interpeter"
-    def: >
-      A program that runs other programs interactively.
-
+ - slug: interpreter
```

Note to past Zhian: YAML is not a markdown list.

```suggestion
- slug: interpreter
```